singularity/debian/singularityfile_debian

```diff
@@ -0,0 +1,149 @@
+Bootstrap: docker
+From: debian:bullseye
+
+%help
+Singularity container for GRASS GIS to be run into GRASS main directory
+
+%labels
+  Maintainer Luca Delucchi
+
+%setup
+  mkdir /tmp/grass_build
+
+%files
+  . /tmp/grass_build
+
+%post
+    # Install useful libraries
+    apt-get -y update
+    apt-get -y install \
+        build-essential \
+        bison \
+        bzip2 \
+        cmake \
+        curl \
+        flex \
+        g++ \
+        gcc \
+        gdal-bin \
+        gettext \
+        git \
+        libbz2-dev \
+        libcairo2 \
+        libcairo2-dev \
+        libcurl4-gnutls-dev \
+        libfftw3-bin \
+        libfftw3-dev \
+        libfreetype6-dev \
+        libgdal-dev \
+        libgeos-dev \
+        libgsl-dev \
+        libjpeg-dev \
+        libjsoncpp-dev \
+        libopenblas-base \
+        libopenblas-dev \
+        libnetcdf-dev \
+        libncurses5-dev \
+        libopenjp2-7 \
+        libopenjp2-7-dev \
+        libpdal-dev \
+        libpnglite-dev \
+        libpq-dev \
+        libpython3-all-dev \
+        libreadline-dev \
+        libsqlite3-dev \
+        libtiff-dev \
+        libzstd-dev \
+        make \
+        mesa-common-dev \
+        moreutils \
+        ncurses-bin \
+        netcdf-bin \
+        pdal \
+        python3 \
+        python3-dateutil \
+        python3-dev \
+        python3-gdal \
+        python3-magic \
+        python3-numpy \
+        python3-pdal \
+        python3-pil \
+        python3-pip \
+        python3-ply \
+        python3-requests \
+        python3-setuptools \
+        python3-venv \
+        software-properties-common \
+        sqlite3 \
+        subversion \
+        unzip \
+        vim \
+        wget \
+        zip \
+        zlib1g-dev
+
+    # Set Python 3 as default Python version
+    update-alternatives --install /usr/bin/python python /usr/bin/python3 1
+    echo LANG="en_US.UTF-8" > /etc/default/locale
+    # compile GRASS
+    cd /tmp/grass_build
+    # -O2 -march=native -std=gnu99 -m64
+    LDFLAGS="-s"
+    CFLAGS="-O2 -march=x86-64 -std=gnu99 -m64"
```

Comment:
Not sure if specifying `-march=x86-64` is a good idea

Reply:
Removed